code/game/turfs/turf.dm

```diff
@@ -32,7 +32,7 @@ GLOBAL_LIST_EMPTY(station_turfs)
 	var/temperature = T20C
 	///Used for fire, if a melting temperature was reached, it will be destroyed
 	var/to_be_destroyed = 0
-	///The max temperature of the fire which it was subjected to
+	///The max temperature of the fire which it was subjected to, determines the melting point of turf
 	var/max_fire_temperature_sustained = 0
```

Comment:
This documentation is now wrong, you should just make a new variable with a new name you're carrying too much baggage here